src/agent/internals/src/test/java/com/google/devtools/cdbg/debuglets/java/FirebaseClientTest.java

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
Nit: Technically this isn't part of this set of changes.

Reply:
Update removed.